commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/CommerceTableTag.java

```diff
@@ -215,15 +216,34 @@ protected int getMinPageSize() {
 	private void _setItems(String dataProviderKey) throws Exception {
 		Map<String, Object> context = getContext();
 
+		String tableName = GetterUtil.getString(context.get("tableName"));
+
+		if (tableName.equals(ParamUtil.getString(request, "tableName"))) {
+			int pageNumberFromURL = ParamUtil.getInteger(request, "pageNumber");
```

Comment:
Rename this variable to `pageNumber` -- we can do that now that we're removing the other `pageNumber` var within this scope.